### PR TITLE
Fixes #546

### LINKED ISSUE #546: Gap between hierarchical node's boundary and edge's start point
Consider the following graph:
 https://rtsys.informatik.uni-kiel.de/elklive/models.html?link=realworld%2Fptolemy%2Fhierarchical%2Falgebraic_heateropentank_HeaterOpenTank.json&owner=eclipse&repo=elk-models

While there are no gaps between any hierarchical node's boundary and the start point of an edge, just that is the case when looking at the result of the layout graph view with the current master. 

image

### PR DESCRIPTION
**Pull request by le-cds**
Fixes both #546, #530.

### CODE REVIEW

File: docs/content/documentation/tooldevelopers/graphdatastructure/layoutoptions.md [hunk 1]
```diff
@@ -90,6 +90,17 @@ This means that you can define multiple `IProperty` instances that describe the
 That is the reason why the first example above does not use `CoreOptions.DEBUG_MODE` to retrieve the property value, but `MyAlgorithmOptions.DEBUG_MODE` (where `MyAlgorithmOptions` is the metadata class generated for your layout algorithm).
 
 
+### Checking for the Presence of Property Values
+
+If we can always call `getProperty(...)` on a graph element, we may not be able to distinguish a default value from a value explicitly set on that element. If we need to, we can check whether a property was explicitly set through `hasProperty(...)`:
```
Comment (uruuru) [inline docs/content/documentation/tooldevelopers/graphdatastructure/layoutoptions.md:95]: The first _can_ feels a bit awkward to me. Maybe rephrase?

**Comment by uruuru** [comment]
It seems there's a trend towards single-line fixes for rather complicated-looking issues :).

File: docs/content/documentation/tooldevelopers/graphdatastructure/layoutoptions.md (hunk 1 above)
Comment (le-cds) [reply docs/content/documentation/tooldevelopers/graphdatastructure/layoutoptions.md]: Done, thanks. :)

**Comment by le-cds** [comment]
> It seems there's a trend towards single-line fixes for rather complicated-looking issues :).

I choose to interpret that as evidence of how well the algorithms already work in general. 😄
